docs/content/en/docs/concepts/clusterworkflow.md

```diff
@@ -8,8 +8,9 @@ description: >
 ---
 
 Each EKS Anywhere cluster is built from a cluster specification file, with the structure of the configuration file based on the target provider for the cluster.
-Currently, Bare Metal, CloudStack, and VMware vSphere are the recommended providers for supported EKS Anywhere clusters.
-We step through the cluster creation workflow for those providers here.
+Currently, Bare Metal, CloudStack, Nutanix, Snow, and VMware vSphere are the recommended providers for supported EKS Anywhere clusters.
+Docker is available as an unsupported provider.
```

Comment:
We also support docker

Reply:
I think the wording is correct: "Docker is available as an unsupported provider."